src/project-css/app/Http/Controllers/CollectionController.php

```diff
@@ -51,36 +54,94 @@ public function index() {
     return view('admin/collection')->with('collcntNms', $collcntNms);
   }
 
+  /**
+  * Returns view for collection
+  *
+  * @return view
+  */    
+  public function show($colID) {
+    // find the collection
```

Comment:
Does this return a view model?  View is vague we need a better description here.